frontend/src/routes/Credentials/CredentialsForm.tsx

```diff
@@ -949,10 +951,20 @@ export function CredentialsForm(props: {
                     patch.push({ op: 'replace', path: `/stringData`, value: secret.stringData })
                 }
                 return patchResource(secret, patch).promise.then(() => {
+                    toastContext.addAlert({
+                        title: t('credentialsForm.updated.title'),
+                        type: 'success',
+                        autoClose: true,
+                    })
                     history.push(NavigationPath.credentials)
                 })
             } else {
                 return createResource(stateToData() as IResource).promise.then(() => {
+                    toastContext.addAlert({
+                        title: t('credentialsForm.created.title'),
+                        type: 'success',
+                        autoClose: true,
+                    })
```

Comment:
A small enhancement might be to add which Credential was actually created/updated?

Reply:
Could, but they just finished creating or editing it. I feel it is enough just to give feedback that it was created.

Reply:
Checked with Joy, she seems to like including the name.  We should probably standardize the toasts across this, cluster pools, and discovered clusters, so maybe:

Title: Credential created
Message: james-credential-1 was successfully created.